bin/periphery/src/docker.rs

```diff
@@ -31,7 +31,8 @@ impl Default for DockerClient {
   fn default() -> DockerClient {
     DockerClient {
       docker: Docker::connect_with_local_defaults()
```

Comment:
What about trying this method?
```rs
Docker::connect_with_defaults()
        .expect("failed to connect to docker daemon"),
```

Reply:
I changed that and ran and tested and that works the same as the previous setup. Is it better code practice? I'm not familiar with Rust.

Reply:
Just a slightly different behavior. 

Doing `connect_with_local_defaults` first may succeed to connect to some local docker socket, even if the user passes `DOCKER_HOST=http://..`, as this method will fallback DOCKER_HOST to `unix:///var/run/docker.sock` if you pass an http address: https://github.com/fussybeaver/bollard/blob/713f07f4fe29bd92a5326379e925504f80fad6d9/src/docker.rs#L906. 

So it would only use `connect_with_http_default` as a fallback if that connection fails. Using `connect_with_defaults` will ensure it will use `DOCKER_HOST` even in the case that a local socket is available, as the user would expect.

Reply:
Thanks for the explanation. I couldn't find that function in the bollard docs, but what you're saying does make sense. I'll modify the pr.